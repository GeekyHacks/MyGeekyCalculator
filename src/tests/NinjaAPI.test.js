import React from 'react';
import { render } from '@testing-library/react';
import QuoteApi from '../components/NinjaAPI';

test('FetchedApi renders correctly', () => {
  const { container } = render(<QuoteApi />);
  expect(container).toMatchSnapshot();
});
