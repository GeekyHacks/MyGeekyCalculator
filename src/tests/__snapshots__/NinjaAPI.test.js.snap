// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FetchedApi renders correctly 1`] = `
<div>
  <div>
    Loading...
  </div>
</div>
`;
